### ISSUE TITLE
Dimmer Percentage doesn't update correctly in Homekit

### ISSUE THREAD

**Issue by Hooger0000** [basdelfos/homebridge-tuya-web#47]
I'm using the newer Treatlife dimmer.  I can adjust the dimmer, and the actual light will stay at the dimmer setting, but after a few seconds, the dimmer shows back at 100% on Homekit, even though the actual light is staying at the set dimmer setting. 

Also, it is showing color options when this switch doesn't offer this.

All in all, it works but is showing wrong dimmer settings and offering color options

**Comment by uberfunk** [comment]
This was addressed in Issue #23  . 
tl;dr: you need to set the `device_type` to `dimmer`, as mentioned in the [Overrules/Defaults section](https://github.com/basdelfos/homebridge-tuya-web#overrule--default-values ) of the main page.

**Comment by Hooger0000** [comment]
> This was addressed in Issue #23 .
> tl;dr: you need to set the `device_type` to `dimmer`, as mentioned in the [Overrules/Defaults section](https://github.com/basdelfos/homebridge-tuya-web#overrule--default-values) of the main page.

Thank you this worked. The dimmer is now showing the correct percentage. Is there a way to get rid of the color options?
